js/index.js

```diff
@@ -1,7 +1,43 @@
 // Iteration 1: Names and Input
 
+let name1= "pablo";
+let name2= "elizabeth";
+console.log(name2);
```

Comment:
En esta parte se pedía imprimir `"The driver's name is XXXX"` y `"The navigator's name is YYYY"`. Aunque estoy seguro que sabéis hacerlo, no os olvidéis de ponerlo.